docs/source/running_mypy.rst

```diff
@@ -439,7 +439,7 @@ This is computed from the following items:
 
 .. note::
 
-    You cannot point to a :pep:`561` package via the ``MYPYPATH``, it must be
+    You cannot point to a stub-only package via the ``MYPYPATH``, it must be
```

Comment:
Can you change this to something like
```
stub-only (:pep:`561`) package
```
so that there will be further information available via the link?

Reply:
Good point! I re-introduced the link.